### ISSUE TITLE
terraform apply crashes when creating aws_ssm_activation with an expiration_date

### ISSUE THREAD

**Issue by vatervonacht** [terraform-providers/terraform-provider-aws#3594]
### Terraform Version
Terraform v0.11.3
+ provider.aws v1.10.0

### Affected Resource(s)
- aws_ssm_activation

### Terraform Configuration Files
```hcl
resource "aws_ssm_activation" "donkey" {
  name               = "Donkey Car Activation"
  description        = "Activation to manage a Donkey Car fleet"
  iam_role           = "${aws_iam_role.ssm_donkey.id}"
  registration_limit = "5"
  expiration_date    = "2018-03-27"
  depends_on         = ["aws_iam_role_policy_attachment.ssm"]
}
```

### Debug Output
See Panic Output below

### Panic Output
https://gist.github.com/vatervonacht/4c4755b3a6667b75a31c59ba3003caec

### Expected Behavior
SSM Activation resource should have been created with the specified `expiration date`

### Actual Behavior
Crashed - stack trace says something about a string being provided instead of time.Time

### Steps to Reproduce

1. `terraform apply`

### Important Factoids
Running on Mac OS High Sierra 10.13.3

### References
None

**Comment by bflad** [comment]
Looks like this attribute was never properly being acceptance testing to catch this! 😢  I'll add a failing then passing test. More soon.

**Comment by bflad** [comment]
Fix PR submitted: #3597

**Comment by bflad** [comment]
The fix for this has been merged into master and will release with v1.11.0 of the AWS provider, likely next week.

**Comment by bflad** [comment]
This has been released in [version 1.11.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1110-march-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
